File: LeetCode/1.Two-Sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    if (nums[0] + nums[1] === target) {
        list = [0,1];
        return list
    } else if (nums[1] + nums[2] === target) {
        list = [1,2];
        return list
    } else if (nums[0] < 0) {
        const tamanho = nums.length -2
        list = [0,tamanho]
        return list
    } else {
        const tamanho = nums.length - 1
        list = [0,tamanho];
        return list
    }
};


console.log(twoSum([3,2,3,2],0))

File: LeetCode/Python/1. Two Sum.py
class Solution(object):
    def twoSum(self, nums, target):
        tamanho = len(nums)
        if tamanho > 2: 
            soma2 = nums[1] + nums[2]
        else:
            soma = nums[0] + nums[1]
            if soma == target:
                return [0,1]
            elif soma2 == target:
                return [1,2]
            else:
                return [0,1]

if __name__ == '__main__':
    nums = Solution()
    nums.twoSum([2,7,11,15],9)